matchers/caseInsensitive/todomatcher.go

```diff
@@ -0,0 +1,39 @@
+package caseInsensitive
```

Comment:
name should be `case_insensitive` as that's the convention for package names in go.